### DISCUSSION TITLE
issue with next/script

### DISCUSSION THREAD

**Discussion by XahidEx** [vercel/next.js#39833] [Help]
"next/script" script components only work on Desktop mode. Not on the mobile version. 
I'm using AdSense on my NextJs blog. When I used script components, I get "99-100" score (pagespeed.web.dev/) but on mobile, It's 50-65. Point to remember that I've tried all script components for testing. Without this script, I get 90+ scores on mobile. 
What can be the problem?

**Comment by icyJoseph** [comment]
I'd be a bit careful with using 3rd party measurement's. 

I am also assuming that you mean, `only work on Desktop mode` as a way of saying that it gives good scores on desktop, but not on mobile, and that the script actually do work on any platform.

When running mobile performance measurements, tools often set their standards very high. I do remember a tool that required a max of 3 script tags per page. 

For example take this article, https://www.webnots.com/adsense-vs-page-speed-an-irony/, while we may agree or not with it, we can ask ourselves, if we made a large site manually, and added the adsense script, would we be penalized? most certainly yes, which means in turn that the framework we use is really not at fault.

**Comment by XahidEx** [comment]
My blog is not heavy. And as I said. Without this script, I always got 93+ scores. Even on the mobile version. But now It's not like before. I even tried LazyLoading 🤧
![20220823_030851](https://user-images.githubusercontent.com/88249321/186019539-5cc8d88c-17c1-4f18-957f-38529c8ca570.png)
![20220823_030905](https://user-images.githubusercontent.com/88249321/186019556-0ef7523c-816f-45ab-ab69-5f622024e72e.png)
